src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicEncapsulateField.cs

```diff
@@ -34,11 +35,11 @@ public void EncapsulateThroughCommand()
             var encapsulateField = VisualStudio.EncapsulateField;
             var dialog = VisualStudio.PreviewChangesDialog;
             encapsulateField.Invoke();
-            dialog.VerifyOpen(encapsulateField.DialogName);
+            dialog.VerifyOpen(encapsulateField.DialogName, timeout: TimeSpan.FromSeconds(10));
```

Comment:
Is this just a randomly chosen timeout? We had _generally_ wanted to avoid random activity timeouts like this because this is what created a lot of flakiness in the _last_ integration test framework, but I don't mind this for now as a way to chase down what's going on.

Reply:
➡️ Extended to 1 minute and refactored to a documented shared constant